client/checkout/classic/index.js

```diff
@@ -18,6 +19,7 @@ jQuery( function ( $ ) {
 	enqueueFraudScripts( getConfig( 'fraudServices' ) );
 
 	const publishableKey = getConfig( 'publishableKey' );
+	const paymentIntent = getConfig( 'paymentIntent' );
```

Comment:
Initially, I used an AJAX request to create the payment intent to return the client secret and intent ID for the UPE, but here I decided to include both fields in the config instead, along with the Stripe key. I thought this was a bit cleaner and smoother on the front-end, but feel free to chirp me if there might be some issues with this approach!

Reply:
I like where you were heading with that move. One thing I just checked and would want to confirm is if the `create_payment_intent` call in `payment_fields` blocks the initial checkout page load. In my test, it seems to which isn't a big deal necessarily. The other thing I noticed though is that then when the payments area renders, it calls `payment_fields` again, which creates a new intent. So every checkout page load may likely create 2 intents. Before changing anything back, I was hoping you could confirm this behavior? If it is holding up the checkout page load and creating 2 intents, I would lean towards the AJAX approach to solve both issues. If it's only one or the other, we could look into solutions for that instead.

Reply:
I actually liked that it blocked and furthermore broke the gateway if it couldn't create the payment intent, but I just tested by injecting a `console.log` statement inside `payment_fields` and it looks like it is firing twice. I always say I'll try anything twice, but creating payment intents for a checkout is where I draw the line. 

I'll revert this to the AJAX way and I'll keep it non-blocking for now, though we may want to reconsider this. The previous card element's unmounted CSS used a fake input to stall, while the Stripe element mounted: we won't be able to do that with all the UPE fields, so it will just flash into being. Perhaps this won't be a bad thing.

Reply:
> so it will just flash into being

Good call on this! Maybe we can find a workaround if the popping in is super noticeable. If it does seem problematic, we can handle it in a separate issue.